calico_containers/calico_ctl/profile.py

```diff
@@ -57,12 +57,91 @@
 """
 import sys
 import re
+import netaddr
+from netaddr import AddrFormatError
 from prettytable import PrettyTable
 from pycalico.datastore import Rule
 from pycalico.datastore import Rules
 from utils import client
 from utils import print_paragraph
 
+
+def validate_arguments(arguments):
+    """
+    Validate argument values:
+        <PROFILE>
+        <SRCTAG>
+        <SRCCIDR>
+        <DSTTAG>
+        <DSTCIDR>
+        <ICMPTYPE>
+        <ICMPCODE>
+
+    Arguments not validated:
+        <SRCPORTS>
+        <DSTPORTS>
+        <POSITION>
+
+    :param arguments: Docopt processed arguments
```

Comment:
I like these comments you put in here for each of the validate functions.  Nice addition